AletheiaFact/aletheia
# Correction bug in layout mobile version

# Description
Addiction of the style for correction bug in layout, text exceed the limit div in the personality page on report in mobile version, 

Fixes #1413 

# Type of change


# Testing
1. Access the mobile version.
2. Create an assertion linked to a personality.
3. Fill in all form fields.
4. Use a filling style without spaces and include as many words as possible, or enter random words until the form field exceeds the page dimensions.
5. Confirm the report to view the preview.

Before:
![Bug layout](https://github.com/user-attachments/assets/c6d2ade2-932b-49c2-9f5a-0729f6661d4a)
ore: 

After:
![bug](https://github.com/user-attachments/assets/fd6368a3-1233-4d54-a203-05b3d09d1966)
![bug mobile](https://github.com/user-attachments/assets/f2211c4e-fc15-41d5-957f-cfce891b57a8)


# Developer Checklist
## General

## Frontend Changes

# Merge Request Review Checklist
